### PR TITLE
Rename to Protocolink

### PR DESCRIPTION
**Pull request by zodahu**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,22 +1,21 @@
...
+Protocolink is a router system proficient in processing ERC20, NFT, and lending positions, providing users and developers greater flexibility when creating transactions across different protocols.
 
-The flexibility of Composable Router comes from its protocol-agnostic nature, with all protocol-related code defined in the below repos and outside of contracts:
+The flexibility of Protocolink comes from its protocol-agnostic nature, with all protocol-related code defined in the below repos and outside of contracts:
 
-- https://github.com/dinngo/composable-router-logics
-- https://github.com/dinngo/composable-router-sdk
-- https://github.com/dinngo/composable-router-api-sdk
+- https://github.com/dinngo/protocolink-logics
+- https://github.com/dinngo/protocolink-sdk
```
Comment (zodahu) [inline README.md:14]: The locations for these two repositories have been confirmed; the api-sdk will be placed inside the sdk.

File: README.md [hunk 2]
```diff
@@ -1,22 +1,21 @@
...
+Protocolink is a router system proficient in processing ERC20, NFT, and lending positions, providing users and developers greater flexibility when creating transactions across different protocols.
 
-The flexibility of Composable Router comes from its protocol-agnostic nature, with all protocol-related code defined in the below repos and outside of contracts:
+The flexibility of Protocolink comes from its protocol-agnostic nature, with all protocol-related code defined in the below repos and outside of contracts:
 
-- https://github.com/dinngo/composable-router-logics
-- https://github.com/dinngo/composable-router-sdk
-- https://github.com/dinngo/composable-router-api-sdk
+- https://github.com/dinngo/protocolink-logics
+- https://github.com/dinngo/protocolink-js-sdk
```
Comment (zodahu) [inline README.md:14]: The locations for these two repositories have been confirmed; the api-sdk will be placed inside the js-sdk.

**Comment by ksin751119** [comment]
LGTM

**Comment by JavanC** [comment]
LGTM

**Pull request merged**